lib/travis.js

```diff
@@ -52,15 +54,24 @@ function travisPublish ({
   const appSlug = appManifestObj.slug
   const appType = appManifestObj.type || 'webapp'
 
+  // get application version to publish
+  let appVersion = ''
+  if (TRAVIS_TAG) {
+    appVersion = TRAVIS_TAG
+  } else {
+    appVersion = `${appManifestObj.version}-dev.${TRAVIS_COMMIT}`
```

Comment:
ici on utilise que `TRAVIS_COMMIT` alors que [la](https://github.com/cozy/cozy-app-publish/pull/14/files#diff-9ac66dc7df8962e9ec8c85dec31ed8baR70) on utilise soir `BUILD_COMMIT || TRAVIS_COMMIT`, je comprends pas pourquoi tu aurais une explication ?

Reply:
It's better to track the archive version using the commit in charge of the publishing. If we use the BUILD_COMMIT, we will give to the version the commit of the build but not the real commit which ran the travis job in charge of the publishing.